Flow.Launcher/Converters/TextConverter.cs

```diff
@@ -11,14 +11,19 @@ public class TextConverter : IValueConverter
         public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
         {
             var id = value?.ToString();
-            return id switch
+            var translationKey = id switch
             {
-                PluginStoreItemViewModel.NewRelease => InternationalizationManager.Instance.GetTranslation("pluginStore_NewRelease"),
-                PluginStoreItemViewModel.RecentlyUpdated => InternationalizationManager.Instance.GetTranslation("pluginStore_RecentlyUpdated"),
-                PluginStoreItemViewModel.None => InternationalizationManager.Instance.GetTranslation("pluginStore_None"),
-                PluginStoreItemViewModel.Installed => InternationalizationManager.Instance.GetTranslation("pluginStore_Installed"),
-                _ => id
+                PluginStoreItemViewModel.NewRelease => "pluginStore_NewRelease",
+                PluginStoreItemViewModel.RecentlyUpdated => "pluginStore_RecentlyUpdated",
+                PluginStoreItemViewModel.None => "pluginStore_None",
+                PluginStoreItemViewModel.Installed => "pluginStore_Installed",
+                _ => null
```

Comment:
I think (not sure) you can just return here.

Reply:
I don't think I can.
![Screenshot](https://github.com/Flow-Launcher/Flow.Launcher/assets/3993179/f3cb5a7e-713e-4602-b61e-efdd3931ec5e)

Reply:
Do you want me to use the switch statement here instead?

Reply:
no it's ok